packages/myst-cli/src/build/utils/collectExportOptions.ts

```diff
@@ -16,6 +23,30 @@ import { getDefaultExportFilename, getDefaultExportFolder } from './defaultNames
 
 export const SOURCE_EXTENSIONS = ['.ipynb', '.md', '.tex'];
 
+function assignArticlesFromProject(exp: Export, proj: Omit<LocalProject, 'bibliography'>) {
+  const { file, pages } = proj;
+  if (MULTI_ARTICLE_EXPORT_FORMATS.includes(exp.format)) {
+    exp.articles = [{ file, level: 1 }, ...pages];
+  } else {
+    exp.articles = [{ file }];
+  }
+  if (exp.format === ExportFormats.xml) {
+    exp.sub_articles = pages
+      .map((page) => (page as any).file)
+      .filter((pageFile): pageFile is string => !!pageFile);
+  }
+}
+
+function assignArticlesFromTOC(session: ISession, exp: Export, tocPath: string) {
+  const allowLevelLessThanOne = [
+    ExportFormats.tex,
+    ExportFormats.pdf,
+    ExportFormats.pdftex,
+  ].includes(exp.format);
+  const proj = projectFromToc(session, tocPath, allowLevelLessThanOne ? -1 : 1);
```

Comment:
tex/pdf exports may have part/chapter with levels -1 / 0 respectively. This also needs to take into account the template, though. These levels should only be used for book templates.